### PR TITLE
Update b2b browse docs

### PR DESCRIPTION
**Pull request by markbackhouse**
### How the new grids look (looking for thoughts on this slightly changed appearance as well as sanity-checking the values 😉):

image

### CODE REVIEW

File: source/includes/_flights_browse_dates.md [hunk 1]
```diff
@@ -50,11 +50,11 @@ GET `/browsedates/v1.0/{country}/{currency}/{locale}/{originPlace}/{destinationP
 
 The table below shows the level of precision supported for the origin and destination places. For more information on these, see [places](#origin-destination-place-in-browse-requests).
 
-![diagram](/images/browsedates_places.png)
+![diagram](/images/browsedates_places_updated.png)
```
Comment (edmgt) [inline source/includes/_flights_browse_dates.md:53]: any reason why you want to keep the old images?

File: source/includes/_flights_browse_dates.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ GET `/browsedates/v1.0/{country}/{currency}/{locale}/{originPlace}/{destinationP
 | ```locale``` <br><span class="required">REQUIRED</span> | The [locale](#locales) you want the results in (ISO locale) |
 | ```originPlace``` <br><span class="required">REQUIRED</span> | The origin place (see [places](#origin-destination-place-in-browse-requests)) |
 | ```destinationPlace``` <br><span class="required">REQUIRED</span> | The destination place (see [places](#origin-destination-place-in-browse-requests)) |
-| ```outboundPartialDate``` <br><span class="required">REQUIRED</span> | The outbound date. Format "yyyy-mm-dd", "yyyy-mm" or "anytime". |
-| ```inboundPartialDate``` <br><span class="optional">OPTIONAL</span> | The return date. Format "yyyy-mm-dd", "yyyy-mm" or "anytime". Use empty string for oneway trip. |
+| ```outboundPartialDate``` <br><span class="required">REQUIRED</span> | The outbound date (see [dates](#outbound-inbound-date-specificity-in-browse-requests)) |
```
Comment (edmgt) [inline source/includes/_flights_browse_dates.md:41]: I'm not sure how this works but shouldn't the link be specific to each endpoint?

File: source/includes/_flights_browse_dates.md (hunk 1 above)
Comment (markbackhouse) [reply source/includes/_flights_browse_dates.md:53]: Not really... to be honest this repo has a **lot** of old files just lying around that aren't used 🙈 I can remove these though

File: source/includes/_flights_browse_dates.md (hunk 2 above)
Comment (markbackhouse) [reply source/includes/_flights_browse_dates.md:41]: No the place it links to isn't part of each endpoint's page. It's on its own page.

File: source/includes/_flights_browse_dates.md (hunk 1 above)
Comment (edmgt) [reply source/includes/_flights_browse_dates.md:53]: 👍

File: source/includes/_flights_browse_dates.md (hunk 2 above)
Comment (edmgt) [reply source/includes/_flights_browse_dates.md:41]: ah cool, makes sense 👍

**Review by edmgt — APPROVED** [review]
nice one, the new images are so much better! and I think they make sense 😄

**Pull request merged**